inline/Cargo.toml

```diff
@@ -18,7 +18,10 @@ harness=false
 
 [dependencies]
 logid.workspace = true
+ribbon.workspace = true
 unimarkup-commons = { path = "../commons/", version = "0" }
+unicode-segmentation = "1.8.0"
```

Comment:
why this dependency?
for Unicode, we should try to stick with icu crates.
The crate builds without it so I guess it is a leftover of your debugging session?

Reply:
This just appeared out of thin air. Absolutely no idea where it came from 🤣. 
There was no intent to use something other than `icu`, so will remove this of course!